### PR TITLE
Fix PR links in 3.2 CHANGELOG

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 * Fix: Avoid open-ended dependencies for sinatra-contrib and rack-protection ([#1949])
 
-* Fix: Helpful message when `Sinatra::Runner` times out ([#1975])
+* Fix: Helpful message when `Sinatra::Runner` times out ([#1960])
```
Comment (dentarg) [inline CHANGELOG.md:17]: This is not correct

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,7 +23,7 @@
 [#1949]: https://github.com/sinatra/sinatra/pull/1949
 [#1952]: https://github.com/sinatra/sinatra/pull/1952
 [#1960]: https://github.com/sinatra/sinatra/pull/1960
-[#1975]: https://github.com/sinatra/sinatra/pull/1960
+[#1975]: https://github.com/sinatra/sinatra/pull/1975
```
Comment (dentarg) [inline CHANGELOG.md:26]: This is good though 👍

File: CHANGELOG.md (hunk 1 above)
Comment (dentarg) [reply CHANGELOG.md:17]: This did happen in #1975 at https://github.com/sinatra/sinatra/pull/1975/files#diff-27e0daa68dc114ddbf9c51bb075e3ca12a107537259d742b5309e6d9e821e86f

Comment (rnestler) [reply CHANGELOG.md:17]: Ah my mistake. 1960 looked related to `Sinatra`, so I thought there we actually want that link.

Comment (rnestler) [reply CHANGELOG.md]: Fixed.

**Review by dentarg — APPROVED** [review]

**Pull request merged**